resources/wf_builder.rb

```diff
@@ -247,7 +259,9 @@
           platform_version: node['platform_version'],
         }
 
-        runner = Mixlib::ShellOut.new("delivery api post runners \
+        # TODO: Rework this to call the API directly, so we don't have delivery-cli formatting things.
+        runner = Mixlib::ShellOut.new("delivery --non-interactive --no-color \
```

Comment:
Order matters here :( `delivery api post runners --non-interactive --no-color` has the intended behavior but `delivery --non-interactive --no-color api post runners` does not, if you order it like this: https://github.com/chef-cookbooks/chef-ingredient/blob/ad/wf-builder-encoding-fix/resources/wf_builder.rb#L263-L264 I see the desired outcome